x-pack/plugins/lens/public/visualizations/datatable/components/columns.tsx

```diff
@@ -171,20 +176,23 @@ export const createGridColumns = (
           value: rowValue,
           columnMeta,
         };
+
+        if (rowValue === null) {
```

Comment:
@tkajtoch FYI, this isn't a 1:1 conversion of the old code. `!= null` catches undefined **and** null, not just null. The equivalent here would be:

```suggestion
        if (rowValue == null) {
```

Reply:
You're totally right! Muscle memory took over on this one